gobject-base/src/guard.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2016-2017 Sebastian Dröge <sebastian@centricular.com>
+//               2016 Luis de Bethencourt <luisbg@osg.samsung.com>
```

Comment:
This was just copy & paste, Luis didn't touch the src/lib.rs so far so let's not copy&paste it any further